src/search/ScrollTrackMarkers.js

```diff
@@ -188,6 +220,20 @@ define(function (require, exports, module) {
         }
     }
 
+    /**
+     * @return {number} amount of vertical space above and below the scrollbar, in pixels
+     */
```

Comment:
Here and below, can you give a proper description?
Maybe just put the description of the return/param in the description, something like:
```js
/**
 * Vertical space above and below the scrollbar.
 * @return {number} amount Value in pixels
 */
```